lints/lint_old_sub_ca_rsa_mod_less_than_1024_bits.go

```diff
@@ -20,14 +20,12 @@ func (l *subCaModSize) Initialize() error {
 func (l *subCaModSize) CheckApplies(c *x509.Certificate) bool {
 	issueDate := c.NotBefore
 	endDate := c.NotAfter
-	return (util.IsSubCA(c) && issueDate.Before(util.RsaDate2) && endDate.Before(util.RsaDate3))
+	_, ok := c.PublicKey.(*rsa.PublicKey)
+	return (util.IsSubCA(c) && issueDate.Before(util.RsaDate2) && endDate.Before(util.RsaDate3)) && ok
```

Comment:
Please change these to all be of the following form:
`ok && util.IsSubCA(c) && issueDate.Before(util.RsaDate2) && endDate.Before(util.RsaDate3)`